lib/css_parser/rule_set.rb

```diff
@@ -107,7 +110,7 @@ def each_declaration # :yields: property, value, is_important
     # TODO: Clean-up regexp doesn't seem to work
     #++
     def declarations_to_s(options = {})
-     options = {:force_important => false}.merge(options)
+     options[:force_important] ||= false
```

Comment:
hmm this now modifies the passed in options, whereas before they were not modified ... check the callers to make sure that's ok or use `.dup`

Reply:
Hmm. Tests passed, so I assumed this would be ok, but it's probably better to use the safer `dup`. Will update.

Reply:
I'm fine with modifying if it's an internal method ... but yeah might come back to bit us :D